packages/jest-validate/src/__tests__/fixtures/jestConfig.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+import type {DefaultConfig, Config} from 'types/Config';
```

Comment:
Do we need this to be typed properly? I kind of don't want to have to maintain the correct types here.